From 369d3ba3902b0c50951ad79ac728c5c9b70af602 Mon Sep 17 00:00:00 2001
From: Mike Looijmans <mike.looijmans@topic.nl>
Date: Thu, 20 Feb 2014 18:13:49 +0100
Subject: [PATCH 6/7] Set SPL offset in qspi flash

To allow existing images to boot, set the offset to 64k, so the second
sector contains u-boot.img. Otherwise there isn't enough room because
the environment is stored at 0x60000.
---
 include/configs/zynq-common.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/zynq-common.h b/include/configs/zynq-common.h
index 7dd734f..9e6085b 100644
--- a/include/configs/zynq-common.h
+++ b/include/configs/zynq-common.h
@@ -415,7 +415,7 @@
 #define CONFIG_SPL_SPI_LOAD
 #define CONFIG_SPL_SPI_FLASH_SUPPORT
 #define CONFIG_SPL_SPI_BUS	0
-#define CONFIG_SYS_SPI_U_BOOT_OFFS	0x100000
+#define CONFIG_SYS_SPI_U_BOOT_OFFS	0x10000
 #define CONFIG_SPL_SPI_CS	0
 #endif
 
@@ -434,7 +434,7 @@
 #define CONFIG_SPL_TEXT_BASE	0x0
 
 /* 3 * 64kB blocks of OCM - one is on the top because of bootrom */
-#define CONFIG_SPL_MAX_FOOTPRINT	0x30000
+#define CONFIG_SPL_MAX_FOOTPRINT	0x10000
 
 /* The highest 64k OCM address */
 #define OCM_HIGH_ADDR	0xffff0000
-- 
1.7.9.5

